frontend/apps/reader/modules/readerrolling.lua

```diff
@@ -55,7 +55,8 @@ local ReaderRolling = InputContainer:new{
     -- With visible_pages=2, in 2-pages mode, ensure the first
     -- page is always odd or even (odd is logical to avoid a
     -- same page when turning first 2-pages set of document)
-    odd_or_even_first_page = 1 -- 1 = odd, 2 = even, nil or others = free
+    odd_or_even_first_page = 1, -- 1 = odd, 2 = even, nil or others = free
+    hide_nonlinear = nil,
```

Comment:
`hide_nonlinear_flows` may be, to make it clearer it's related to "flows" elsewhere.